### PR TITLE
use webtorrent fixtures

### PR DESCRIPTION
**Pull request by rom1504**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -16,6 +16,7 @@
     "brfs": "^1.0.0",
     "standard": "^6.0.5",
     "tape": "^4.0.0",
+    "webtorrent-fixtures": "rom1504/webtorrent-fixtures#add_parse_torrent_file_fixtures",
```
Comment (rom1504) [inline package.json:19]: This should be moved back to a version once the PR in webtorrent-fixtures is merged and released. Just for testing.

Comment (rom1504) [inline package.json:19]: depends on https://github.com/feross/webtorrent-fixtures/pull/1

**Comment by rom1504** [comment]
tests are failing because of missing saucelabs credentials but they do pass in node

**Comment by rom1504** [comment]
@feross well at least some of the torrent in webtorrent-fixtures can directly be used here without adding more torrents to it. For example leaves and corrupt.
I thought it would be useful to put all the variety of example torrents in webtorrent-fixtures to make it easier to test torrent modules.

I can understand that keeping the parsed torrent here might make more sense.

**Comment by feross** [comment]
@rom1504 You're right - I went ahead and made the following changes: 
- reuse `leaves`, `corrupt`, and `leavesMetadata` (i.e. `leaves-magnet`)
- get rid of `pride` and just use `numbers` as the multi-file torrent

Changes here: https://github.com/feross/parse-torrent-file/commit/7604a548245871da239f5eabd034d9046a8363f7 and https://github.com/feross/parse-torrent-file/commit/0bf9adf152140ed046d3bcac2e89e15e7fe607fd

**Comment by feross** [comment]
Thanks for starting things off with your PR!

**Pull request closed without merging**